### PR TITLE
Add Detailed Description for Check Broken URLs in `CONTRIBUTING.md` and Improve Markdown in 01 and 02

### PR DESCRIPTION
**Pull request by skytin1004**
## Purpose

In this PR, I've added a detailed description in `CONTRIBUTING.md` explaining how the "Check Broken URLs" check in Markdown Validation works and what to do if an issue is detected. Additionally, I've improved the markdown in `01` and `02` files and removed misplaced photo files in `02`.

### Changes
1. **CONTRIBUTING.md:**
   - Added a comprehensive section detailing the "Check Broken URLs" validation process.
   - Included instructions on how to address issues if the check fails.

2. **Markdown Improvements:**
   - Enhanced the formatting and structure of markdown files `01` and `02` for better readability and consistency.
   - Corrected any existing formatting issues.

3. **File Cleanup:**
   - Removed misplaced photo files from the `02` directory to maintain a clean and organized repository structure.

## Does this introduce a breaking change?

When developers merge from main and run the server, azd up, or azd deploy, will this produce an error?
If you're not sure, try it out on an old environment.

```
[ ] Yes
[x] No
```

## Does this require changes to learn.microsoft.com docs?

This repository is referenced by (https://azure.microsoft.com/products/phi-3)
which includes deployment, settings and usage instructions.

```
[ ] Yes
[x] No
```

## Type of change

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: md/06.E2ESamples/E2E_Phi-3-FineTuning_PromptFlow_Integration.md
Comment (leestott) [inline md/06.E2ESamples/E2E_Phi-3-FineTuning_PromptFlow_Integration.md:1]: Hi please leave this as NC6 as this is the smallest GPU based machine and NC6 capacity is available for FREE subscriptions suggests add a comment that GPU instances and Compute name can be changed and link to MS Docs

**Review by leestott — CHANGES_REQUESTED** [review]
Please resolved the conflict and make suggested change

File: md/06.E2ESamples/E2E_Phi-3-FineTuning_PromptFlow_Integration.md
Comment (leestott) [reply md/06.E2ESamples/E2E_Phi-3-FineTuning_PromptFlow_Integration.md:1]: @skytin1004 thanks for your suggestions please see comments/suggestions

**Comment by skytin1004** [comment]
Hi @leestott, 

I agree with your suggestion about using NC6, as it is indeed the smallest GPU-based machine and NC6 capacity is available for free subscriptions. However, I tested it again yesterday and found that it didn't work well for fine-tuning. Therefore, I’ve updated the tutorial to use Standard_NC24ads_A100_v4 instead.

I will be adding the link to MS Docs soon.  Could you please review it when you have a moment?

Thank you for your feedback

**Comment by skytin1004** [comment]
@leestott I plan to investigate and find a suitable fine-tuning environment specifically for NC6. For now, it seems reasonable to keep the Standard_NC24ads_A100_v4 configuration. If I find a suitable fine-tuning setup for NC6, I'll switch to it.

I’d appreciate your thoughts on this approach. If you believe it's essential to use NC6, please let me know.

**Review by leestott — APPROVED** [review]
Thanks for the contributions and updates.

**Pull request merged**